src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+class RightTriangle implements Figure {
```

Comment:
The class should not start with an empty line. Please remove the empty line above the class declaration.